{"uuid": "aa1dbc67-db86-4f49-8ff9-aedcd1179380", "children": ["9b7723d6-88bc-4f95-a53a-68e638509e49"], "befores": [{"name": "driver_do", "status": "passed", "start": 1721361899648, "stop": 1721361906533}], "afters": [{"name": "driver_do::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\User\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\User\\Diplom\\Diplom_3\\conftest.py\", line 28, in driver_do\n    driver.quit()\n  File \"C:\\Users\\User\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 84, in quit\n    self.service.stop()\n  File \"C:\\Users\\User\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 146, in stop\n    self.send_remote_shutdown_command()\n  File \"C:\\Users\\User\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 126, in send_remote_shutdown_command\n    request.urlopen(f\"{self.service_url}/shutdown\")\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 215, in urlopen\n    return opener.open(url, data, timeout)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 515, in open\n    response = self._open(req, data)\n               ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 532, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 492, in _call_chain\n    result = func(*args)\n             ^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 1373, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\urllib\\request.py\", line 1344, in do_open\n    h.request(req.get_method(), req.selector, req.data, headers,\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 1331, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 1377, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 1326, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 1085, in _send_output\n    self.send(msg)\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 1029, in send\n    self.connect()\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\http\\client.py\", line 995, in connect\n    self.sock = self._create_connection(\n                ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\socket.py\", line 844, in create_connection\n    exceptions.clear()  # raise only the last error\n    ^^^^^^^^^^^^^^^^^^\n"}, "start": 1721362114283, "stop": 1721362122501}], "start": 1721361899648, "stop": 1721362122501}